Gemfile

```diff
@@ -1,6 +1,9 @@
 source 'https://rubygems.org'
 
-gem 'rspec', '~> 3.8.0'
+gem 'rspec', '~> 3.8.0', require: false, group: :test
 gem 'simplecov', require: false, group: :test
 gem 'codecov', require: false, group: :test
+gem 'rack-test', require: false, group: :test
```

Comment:
You can make a `group :test do` yada block to easily group  gems https://bundler.io/v1.3/groups.html